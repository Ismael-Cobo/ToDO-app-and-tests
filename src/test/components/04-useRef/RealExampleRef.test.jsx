const { shallow } = require("enzyme")
const { RealExampleRef } = require("../../../components/04-useRef/RealExampleRef")

describe('prueba del componente <RealExampleRef />', () => {


    test('debe de mostrarse correctamenete', () => {

        const wrapper = shallow(<RealExampleRef />)

        expect(wrapper).toMatchSnapshot()
        expect(wrapper.find('MultipleCustomHooks').exists()).toBe(false)
    })

    test('debe de mostrar el componente <MultipleCustomHooks />', () => {

        const wrapper = shallow(<RealExampleRef />)
        
        const button = wrapper.find('button')

        button.simulate('click')

        expect(wrapper.find('MultipleCustomHooks').exists()).toBe(true)

    })
})