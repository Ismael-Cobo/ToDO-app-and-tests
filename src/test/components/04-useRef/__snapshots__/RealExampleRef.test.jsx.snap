// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`prueba del componente <RealExampleRef /> debe de mostrarse correctamenete 1`] = `
<div>
  <h1>
    RealExampleRef
  </h1>
  <hr />
  <button
    className="btn btn-primary mt-5"
    onClick={[Function]}
  >
    Show/Hide
  </button>
</div>
`;
